cmd/database/create.go

```diff
@@ -74,6 +74,12 @@ For reference, inspect a "metadata.yaml" file generated while running "luet buil
 
 				files := art.Files
 
+				// Check if the package is already present
+				if p, _ := systemDB.FindPackage(art.CompileSpec.GetPackage()); p.GetName() != "" {
```

Comment:
well, I'd say let's start by checking the error before accessing `p`

Reply:
yeah, that's true. i will fix this. thanks